site/docs/reference/Connectors/capture-connectors/facebook-marketing.md

```diff
@@ -0,0 +1,134 @@
+---
+sidebar_position: 5
+---
+
+# Facebook Marketing
+
+This connector captures data from the Facebook Marketing API into Flow collections.
+
+[`ghcr.io/estuary/airbyte-source-facebook-marketing:dev`](https://ghcr.io/estuary/airbyte-source-facebook-marketing:dev) provides the latest connector image. You can also follow the link in your browser to see past image versions.
+
+This connector is based on an open-source connector from a third party, with modifications for performance in the Flow system.
+You can find their documentation [here](https://docs.airbyte.com/integrations/sources/facebook-marketing/),
+but keep in mind that the two versions may be significantly different.
+
+## Supported data resources
+
+The following data resources are supported:
+
+* [Ads](https://developers.facebook.com/docs/marketing-api/reference/adgroup)
+* [Ad activities](https://developers.facebook.com/docs/marketing-api/reference/ad-activity)
+* [Ad creatives](https://developers.facebook.com/docs/marketing-api/reference/ad-creative)
+* [Ad insights](https://developers.facebook.com/docs/marketing-api/reference/adgroup/insights/)
+* [Business ad accounts](https://developers.facebook.com/docs/marketing-api/reference/business/adaccount/)
+* [Campaigns](https://developers.facebook.com/docs/marketing-api/reference/ad-campaign-group)
+* [Images](https://developers.facebook.com/docs/marketing-api/reference/ad-image)
+* [Videos](https://developers.facebook.com/docs/graph-api/reference/video/)
+
+By default, each resource associated with your Facebook Business account is mapped to a Flow collection through a separate binding.
+
+## Prerequisites
+
+* A Facebook Business account, and its Ad Account ID.
+* A Facebook app with:
+  * The [Marketing API](https://developers.facebook.com/products/marketing-api/) enabled.
+  * A Marketing API access token generated.
+  * Access upgrade from Standard Access (the default) to Advanced Access. This allows a sufficient [rate limit](https://developers.facebook.com/docs/marketing-api/overview/authorization#limits) to support the connector.
+
+Follow the steps below to meet these requirements.
+
+### Setup
+
+1. Find your Facebook Business [Ad Account ID](https://www.facebook.com/business/help/1492627900875762).
```

Comment:
Actually, the new Business ID level does not require the AdAccount... but honestly we are changing this as we speak so let's wait on this to see how we end up doing this (we might go back to using a list of AdAccounts)

Reply:
OK, I'll leave it for now but follow up.